File: exs/mundo_1/golang/002.go
/*
	Desafio: 002
	Problema: Crie um script que leia o nome de uma 
	pessoa e mostre uma mensagem de boas-vindas de acordo com 
	o valor digitado
*/
package main
import "fmt"
func main() {
	var name string
	fmt.Println("Olá, qual seu nome?")
	fmt.Scanln(&name)
	fmt.Println("Bem vindo, " + name)
}

File: exs/mundo_1/c/002.c
/*
    002 - Respondendo ao usuário
    Crie um script Python que leia o nome de uma pessoa e mostre uma mensagem de
    boas-vindas de acordo com o valor digitado
*/

#include <stdio.h>

int main(void)
{
    char nome[50];
    printf("Qual o seu nome? ");
    scanf("%s", &nome);
    printf("Seja bem-vindo, %s!\n", nome);
    return 0;
}